src/Tools/IdeCoreBenchmarks/NavigateToBenchmarks.cs

```diff
@@ -85,10 +86,18 @@ public async Task RunNavigateTo()
         private async Task<int> SearchAsync(Project project, ImmutableArray<Document> priorityDocuments)
         {
             var service = project.LanguageServices.GetService<INavigateToSearchService>();
-            var searchTask = service.SearchProjectAsync(project, priorityDocuments, "Document", service.KindsProvided, CancellationToken.None);
-
-            var results = await searchTask.ConfigureAwait(false);
-            return results.Length;
+            var results = new List<INavigateToSearchResult>();
```

Comment:
curious what the difference was (if any) for these benchmarks

Reply:
effectively identical.